### PR TITLE
feat: sign up screen layout

### PR DESCRIPTION
**Pull request by PauloCruz06**

### CODE REVIEW

File: src/components/Signup.js [hunk 1]
```diff
@@ -1,5 +1,121 @@
...
 export default function Signup(){
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [name, setName] = useState("");
+    const [photo, setPhoto] = useState("");
+
+    /*function signUp(e){
+        e.preventDefaut();
+
+    }*/
```
Comment (SauloDriven) [inline src/components/Signup.js]: **Question**: why is this commented?

**Review by SauloDriven — COMMENTED** [review]
a little doubt :smile:

File: src/components/Signup.js (hunk 1 above)
Comment (PauloCruz06) [reply src/components/Signup.js]: I was testing something and I forgot about it there :(

Comment (SauloDriven) [reply src/components/Signup.js]: :cry:

**Review by SauloDriven — APPROVED** [review]
Niceee

**Pull request merged**